Latest update to iot-ds tutorials

anzeiger opened the pull request:
Automatic commit: Move 'iot-ds-1-define-actions', 'iot-ds-2-create-flp', 'iot-ds-3-create-ui', 'iot-ds-4-build-test', 'iot-ds-5-view-ext' from QA to Production
- Update all screenshots and terminologies with SAP Cloud Platform to SAP Business Technology Platform 
- Update outdated iot application screenshots
- Add new steps for iot rule project creation
- Remove obsolete steps

MichaelCzcz commented on tutorials/iot-ds-2-create-flp/iot-ds-2-create-flp.md:
```diff
@@ -4,7 +4,7 @@ description: Create and configure an SAP Fiori launchpad site to receive notific
 time: 40
 primary_tag: topic>internet-of-things
-tags: [ tutorial>beginner, tutorial>license, topic>internet-of-things, topic>cloud, products>sap-leonardo-iot, products>sap-edge-services, products>sap-cloud-platform-internet-of-things, products>sap-cloud-platform, products>sap-cloud-platform-portal, products>sap-web-ide ]
+tags: [ tutorial>beginner, tutorial>license, topic>internet-of-things, topic>cloud, products>sap-edge-services, products>sap-cloud-platform-internet-of-things, products>sap-business-technology-platform, products>sap-web-ide ]
```
products>sap-cloud-platform-internet-of-things -> wrong tag

anzeiger replied:
@MichaelCzcz I have removed the wrong tags and replaced the usage of "we/our" with you/your.  I have also made a change regarding the usage of "we" in the rules.vr file for one of the tutorials.  The change was pushed to QA but some how it didn't get pushed to Production: https://github.com/SAPDocuments/Tutorials-Contribution/commit/382239f577ed57556195f8135540a156cca8bcd9. Can I create a separate PR for this change?

MichaelCzcz replied:
rules.vr is only pushed to QA, but changes will also work in production.

MichaelCzcz commented on tutorials/iot-ds-3-create-ui/iot-ds-3-create-ui.md:
```diff
@@ -4,7 +4,7 @@ description: Create a decision support application using the SAP IoT extension f
 time: 30
 primary_tag: topic>internet-of-things
-tags: [ tutorial>beginner, tutorial>license, topic>internet-of-things, topic>cloud, topic>sapui5, products>sap-leonardo-iot, products>sap-edge-services, products>sap-cloud-platform-internet-of-things, products>sap-cloud-platform, products>sap-web-ide ]
+tags: [ tutorial>beginner, tutorial>license, topic>internet-of-things, topic>cloud, topic>sapui5, products>sap-edge-services, products>sap-cloud-platform-internet-of-things, products>sap-business-technology-platform, products>sap-web-ide ]
```
products>sap-cloud-platform-internet-of-things -> wrong tag

anzeiger replied:
I have removed the wrong tags and replaced the usage of we/our with you/your

MichaelCzcz reviewed (changes requested):
Please check and correct all tags that refer to "cloud platform". I have commented directly on two occurrences, but there are more.
Also, please check and correct all tutorials in this PR for use of "royal we" (e.g.: "We will do this and that."). Don't use "we" unless you are referring to SAP, use "you" instead.

MichaelCzcz commented:
Just removing the cloud platform part isn't enough. Please use the correct tags or the tutorial won't be visible.
I have forwarded the email with Daniel's announcement for new tags to you.

anzeiger commented:
@MichaelCzcz I have removed the irrelevant edge service tag as this service is not used in this tutorial (it was copied from another IoT tutorial when this one was created).  I have updated the product tag for Internet of Things and added sap-fiori to one of the tutorials.

MichaelCzcz commented:
@anzeiger _products>sap-internet-of-things_ ist still an invalid tag. It should be _products>sap-iot-services-for-sap-btp_ as mentioned in the email that I have forwarded to you.
